doctr/models/detection/fast/pytorch.py

```diff
@@ -195,11 +195,16 @@ def forward(
             out["out_map"] = prob_map
 
         if target is None or return_preds:
+            # Disable for torch.compile compatibility
+            @torch.compiler.disable  # type: ignore[attr-defined]
```

Comment:
post proc can't be compiled so model runs fully compiled and all the other parts keept as is